lib/fly.js

```diff
@@ -81,16 +81,28 @@ module.exports = Fly
  * Compose a new, yeildable sequence.
  * Resets instance's glob, filters, and writer.
  *
- * @param  {String|Array} globs The glob selection
+ * @param  {String|Array} ...globs The glob selection
+ * @param {Object} options If the last argument is an object it's used as options to pass `node-glob`
  * @return                      The current Fly instance.
  */
-Fly.prototype.source = function (globs) {
-	globs = arrify(globs)
+Fly.prototype.source = function () {
+	var globs = flatten([].slice.call(arguments))
+	var options = globs.pop()
+
+	if (Object.prototype.toString.call(options) !== '[object Object]') {
```

Comment:
The to string method that I put in there is the only reliable one. If you run this test in the console you should see that what I'm talking about.


```js
var types = {
  number: 1,
  array: [],
  string: '',
  object: {},
  boolean: true
}

for (var key in types) {
  console.log(key)
  var type = types[key]
  console.log("typeof " + key + " !== 'object' = ", typeof type !== 'object')
  console.log(key + ' !== Object(array) =', type !== Object(array))
  console.log('to string method:', Object.prototype.toString.call(type) !== '[object Object]')
  console.log('')
}
```


if `options` is an array and you try to use either of those methods it's going to return `false` when it should return `true`. 

[Here's](http://stackoverflow.com/questions/5876332/how-can-i-differentiate-between-an-object-literal-other-javascript-objects) an issue on stackoverflow about type checking for a plain object